sdks/python/apache_beam/coders/coders_test.py

```diff
@@ -58,6 +59,42 @@ def test_str_utf8_coder(self):
     self.assertEqual('abc', real_coder.decode(real_coder.encode('abc')))
 
 
+class ProtoCoderTest(unittest.TestCase):
+
+  def test_proto_coder(self):
+    ma = test_message.MessageA()
+    mb = ma.field2.add()
+    mb.field1 = True
+    ma.field1 = u'hello world'
+    expected_coder = coders.ProtoCoder(ma.__class__)
+    real_coder = coders.registry.get_coder(ma.__class__)
+    self.assertEqual(expected_coder, real_coder)
+    self.assertEqual(real_coder.encode(ma), expected_coder.encode(ma))
+    self.assertEqual(ma, real_coder.decode(real_coder.encode(ma)))
+
+
+class DummyClass(object):
+  "A class with no registered coder."
+  def __init__(self):
+    pass
+
+  def __eq__(self, other):
+    if isinstance(other, self.__class__):
+      return True
+    return False
+
+
+class FallbackCoderTest(unittest.TestCase):
+
+  def test_default_fallback_path(self):
+    "Test fallback path picks a matching coder if no coder is registered"
```

Comment:
Use """ style quotes for docstrings here and throughout.  Period at end of sentence.